packages/commonjs/test/fixtures/form/multi-entry-module-exports/output1.js

```diff
@@ -0,0 +1,8 @@
+import './input2.js';
+import t2 from '_./input2.js?commonjs-proxy';
+
+console.log(t2);
+var input1 = 1;
+
+export default input1;
+export { input1 as __moduleExports };
```

Comment:
So you said we cannot get rid of `__moduleExports` because it will trigger interop handlers when using the `default` export? The thing is, this will now taint entry points in a way that users will notice there is suddenly a new named export that does not belong here. So one could wonder if for entry points, we might rather use the default even though it triggers interop? But I am undecided, what do you think?

Reply:
Well this only happens for entry points that are being imported by other code paths in the bundle.  
So in most use cases - it will not be exported.  
And in the use cases where it does - it's due to its use by one of the other entry points.

Unfortunately I don't see any way around it without breaking code... But if I think of some other trick to get rid of this, I'll make a PR :-)